openpype/settings/defaults/project_settings/maya.json

```
@@ -39,7 +39,7 @@
         "aov_separator": "underscore",
         "reset_current_frame": false,
         "arnold_renderer": {
-            "image_prefix": "maya/<Scene>/<RenderLayer>/<RenderLayer>_<RenderPass>",
+            "image_prefix": "maya/<Scene>/<RenderLayer>/<RenderLayer>",
```

```suggestion
            "image_prefix": "maya/<Scene>/<RenderLayer>/<RenderLayer>{aov_separator}<RenderPass>",
```

I believe if the setting is like this it'll actually work.

It'll split off everything after the `{aov_separator}` when merge AOVs is enabled and thus should have no separate renderpasses.